README.md

```diff
@@ -0,0 +1,159 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Covid 19 metric</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Covid 19 metric ](#-covid-19-metric-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Covid 19 metric <a name="about-project"></a>
+
+The COVID-19 pandemic, also known as the coronavirus pandemic, is a global pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The novel virus was first identified from an outbreak in Wuhan, China, in December 2019.
+## 🛠 Built With <a name="built-with"></a>
+- React
+- Redux
+- JAVASCRIPT
+- JSX
+-ES6
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/HTML">html5</a></li>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/CSS">CSS3</a></li>
+     <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive Design**
+- **dynamic**
+- **Code Quality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have NodeJS installed
+- A Web Browser (Google Chrome, Firefox, etc)
+- A Code Editor (Notepad++, VSCode, etc)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone your link repo https://github.com/noelfoka/react-capstone-project.git
+
+cd your folder name
+```
+
+### Install
+
+Install this project with:
+
+```sh
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+npx hint .
+
+### Deployment
+
+[Live Demo](https://noel-covid19.onrender.com)
```

Comment:
- [ ] Well done deploying the app. However, it has issues that need to be fixed. Please see the screenshot below and implement the changes. 🙏 
![React-App](https://github.com/noelfoka/react-capstone-project/assets/47625407/8688a285-1144-4347-9c6a-d4cbe7cfe7b1)